api/room_handler_test.go

```diff
@@ -315,8 +328,9 @@ forwarders:
 					"metadata": map[string]interface{}{
 						"ipv6Label": "",
 						"region":    "us",
+						"ports":     "[]",
 					},
-				}, gomock.Any())
+				}, gomock.Any()).AnyTimes()
```

Comment:
It is not ideal to use AnyTimes. If possible assert that HGet is called the exact number of times it should

Reply:
Yeah, it's not needed here. It's only called one time